sktime/transformers/series_as_features/truncation.py

```diff
@@ -18,7 +16,7 @@ class TruncationTransformer(BaseSeriesAsFeaturesTransformer):
     """MyTransformer docstring
```

Comment:
Do you prefer `Truncator` or `TruncationTransformer`?

Reply:
I'm ambivalent about the name. Happy to leave it as is, and we can refactor the name if people don't like it.